travis/publish.sh

```diff
@@ -6,7 +6,7 @@ if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
   docker login --username="$DOCKER_USERNAME" --password="$DOCKER_PASSWORD"
 
   if [ "$TRAVIS_BRANCH" == "master" ]; then
-    if [ "$VERSION" == "12.0" ]; then
+    if [ "$VERSION" >= "12.0" ]; then
```

Comment:
Hum, this means we are going to push the latest with `$VERSION IN [12.0,13.0]` I'm not sure this is what we want.